views/templates/email/waiting_for_payment_adyen.html

```diff
@@ -0,0 +1,804 @@
+                </td>
+                </tr>
+                </table>
+
+                </td>
+                </tr>
+                <![endif]-->
+                <!-- BORDER ENDING -->
+                <!-- SUBTITLE BEGINING -->
+                <!--[if mso | IE]>
+                <tr>
+                    <td
+                            class="" width="604px"
+                    >
+
+                        <table
+                                align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:604px;" width="604"
+                        >
+                            <tr>
+                                <td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;">
+                <![endif]-->
+                <div style="Margin:0px auto;max-width:604px;">
+                    <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;" width="100%"><tbody><tr>
+                        <td style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; direction: ltr; font-size: 0px; padding: 0 25px; text-align: center; vertical-align: top;" align="center">
+                            <!--[if mso | IE]>
+                            <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+
+                                <tr>
+
+                                    <td
+                                            class="" style="vertical-align:top;width:554px;"
+                                    >
+                            <![endif]-->
+                            <div class="mj-column-per-100 outlook-group-fix" style="font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;" align="left" width="100%">
+                                <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt;"><tbody><tr>
+                                    <td style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; vertical-align: top; padding: 0;">
+                                        <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt;" width="100%"><tr>
+                                            <td align="left" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-size: 0px; padding: 10px 25px; padding-top: 0px; padding-bottom: 0px; word-break: break-word;">
+                                                <div style="font-family:Open sans, arial, sans-serif;font-size:16px;font-weight:600;line-height:25px;text-align:left;color:#363A41;" align="left">
+                                                    Order ID {order_name} - Pending payment
```

Comment:
I guess the pending is more like pending on the PSP side while here we are waiting for the customer.
```suggestion
                                                    Order ID {order_name} - Waiting for payment
```

Reply:
https://github.com/Adyen/adyen-prestashop/pull/195/commits/bdbd24f33011e55cbe6df247cc9625b59ae7d9a6